yarn-project/circuit-types/src/logs/l1_payload/l1_event_payload.ts

```diff
@@ -97,23 +85,14 @@ export class L1EventPayload {
    * @returns A random L1EventPayload object.
    */
   static random(contract = AztecAddress.random()) {
-    return new L1EventPayload(Event.random(), contract, Fr.random(), EventSelector.random());
+    return new L1EventPayload(Event.random(), contract, EventSelector.random());
   }
 
   public equals(other: L1EventPayload) {
     return (
       this.event.equals(other.event) &&
       this.contractAddress.equals(other.contractAddress) &&
-      this.randomness.equals(other.randomness) &&
       this.eventTypeId.equals(other.eventTypeId)
     );
   }
 }
-
-function ensureMatchedMaskedContractAddress(contractAddress: AztecAddress, randomness: Fr, maskedContractAddress: Fr) {
```

Comment:
Removing this check from the class because we can't check the siloed field by looking at the payload anymore. We will have to get the "raw" tag and silo it with the expected contract address to confirm that the log is actually from the said contract. And this should be done in components where raw tags are available.